test/scripts/initialize_eks_cluster.bash

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+LD_FLAGS=$1
+EKS_CLUSTER_NAME=$2
+IMAGE_VERSION=$3
```

Comment:
The `IMAGE_VERSION` variable defined in Makefile should be sufficient. Do we need this variable to be passed to the script again as a parameter?

Reply:
No it doesn't need to be passed in. Will remove this

Reply:
Sorry turns out it does need to be passed into the script. 

It's used by [docker push](https://github.com/lasred/secrets-store-csi-driver/blob/master/test/scripts/initialize_eks_cluster.bash#L19) which references $IMAGE_TAG which includes the [$IMAGE_VERSION](https://github.com/lasred/secrets-store-csi-driver/blob/master/test/scripts/initialize_eks_cluster.bash#L13)